modules/nw-view-status-configuration-network-observability-operator.adoc

```diff
@@ -0,0 +1,17 @@
+/ Module included in the following assemblies:
```

Comment:
same here, not sure if it should be visible in the docs?

Reply:
Addressed